### PR TITLE
SOW-24: Entry and start dates for create journey

### PR DESCRIPTION
**Pull request by gcooperSL**
entry and start dates match with the last updated time, for consistency.

[ticket](https://trello.com/c/JDKJL5WV/104-entry-and-start-dates-for-create-journey)

### CODE REVIEW

File: src/pages/export-page/ExportPage.tsx [hunk 1]
```diff
@@ -32,7 +32,13 @@ export const ExportPage = () => {
         developmentPlanEvent === TimetableEventKey.TimetableUpdated
     );
     return [
-      ...timetableEvents,
+      ...timetableEvents.map((x) => {
```
Comment (vferraro-scottlogic) [inline src/pages/export-page/ExportPage.tsx:35]: can we use something meaningful rather than "x"? "Event" maybe?

Comment (gcooperSL) [reply src/pages/export-page/ExportPage.tsx:35]: yep! this was me being lazy and forgot to change it!

**Comment by vferraro-scottlogic** [comment]
We are now setting entryDate and startDate for events before exporting (on the Export page) so I think we don't need to set the dates at the beginning when generating the events? (see getDefaultTimetableEvent)

File: src/pages/export-page/ExportPage.tsx [hunk 2]
```diff
@@ -32,7 +32,13 @@ export const ExportPage = () => {
         developmentPlanEvent === TimetableEventKey.TimetableUpdated
     );
     return [
-      ...timetableEvents,
+      ...timetableEvents.map((Event) => {
```
Comment (vferraro-scottlogic) [inline src/pages/export-page/ExportPage.tsx:35]: lower case "event". My bad I think I had capitalised "Event" in my last comment 😅

**Review by vferraro-scottlogic — APPROVED** [review]

**Pull request merged**